android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -3707,5 +3714,55 @@ public void clear(){
 		 }
 	}
 
-
+	/**
+	 * notifySppError: utilize notification channel to notify the SPP out-of-resource error.
+	 */
+	@TargetApi(11)
+	@SuppressLint("NewApi")
+	private void notifySppError() {
+		Notification.Builder builder;
+		if(android.os.Build.VERSION.SDK_INT < Build.VERSION_CODES.O){
+			builder = new Notification.Builder(getApplicationContext());
+		} else {
+			builder = new Notification.Builder(getApplicationContext(), TransportConstants.SDL_ERROR_NOTIFICATION_CHANNEL_ID);
+		}
+		ComponentName name = new ComponentName(this, this.getClass());
+		if(0 != (getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE)) { //If we are in debug mode, include what app has the router service open
+			builder.setContentTitle("SDL: " + name.getPackageName());
+		} else {
+			builder.setContentTitle(getString(R.string.notification_title));
+		}
+		builder.setTicker(getString(R.string.sdl_error_notification_channel_name));
+		builder.setContentText(getString(R.string.spp_out_of_resource));
+
+		//We should use icon from library resources if available
+		// @REVIEW: large icon uses SPP error specific icon, but can we reuse the small icon with the default tray icon?
```

Comment:
Keeping the current small icon should be fine. Can we remove this comment?

Reply:
I just removed the @REVIEW comment at aac3702 commit. Thanks.